### PR TITLE
Adding in the Container struct for tasks

### PR DESCRIPTION
**Pull request by ShaneSaww**
Adds the attributes for the Container and Docker info on Mesos tasks!

### CODE REVIEW

File: types.go [hunk 1]
```diff
@@ -196,6 +196,30 @@ type Task struct {
...
+// Container represents one way a Mesos task can be ran
+type Container struct {
+	Type   string `json:"type"`
+	Docker Docker `json:"docker,omitempty"`
+}
+
+// Docker is one type of Container
+type Docker struct {
+	Image          string        `json:"image"`
+	Netowrk        string        `json:"network"`
```
Comment (andygrunwald) [inline types.go]: I think this is a typo. `Netowrk ` should be `Network`

**Comment by ShaneSaww** [comment]
Good tach @andygrunwald!
Typo fixed!

File: types.go (hunk 1 above)
Comment (ShaneSaww) [inline types.go]: Fixed!

**Comment by andygrunwald** [comment]
Thanks @ShaneSaww!

**Pull request merged**